README.md

```diff
@@ -42,29 +42,34 @@ Documentation: [https://arduino-esp8266.readthedocs.io/en/2.5.0/](https://arduin
 [![Linux build status](https://travis-ci.org/esp8266/Arduino.svg)](https://travis-ci.org/esp8266/Arduino)
 
 - Install the current upstream Arduino IDE at the 1.8 level or later. The current version is at the [Arduino website](http://www.arduino.cc/en/main/software).
-- Go to Arduino directory
+- Go to your Arduino sketchbook directory
   - For Mac OS X, it is `Arduino.app` showing as the Arduino icon.  
     This location may be your `~/Downloads`, `~/Desktop` or even `/Applications`.
     ```bash
     cd <application-directory>/Arduino.app/Contents/Java
     ```
-  - For Linux, it is ~/arduino by default.
+  - For Linux, it is `~/arduino/` by default.
     ```bash
-    cd ~arduino
+    cd ~/arduino/
     ```
-- Clone this repository into hardware/esp8266com/esp8266 directory (or clone it elsewhere and create a symlink)
+
+- Clone this repository into `hardware/esp8266com/esp8266/` (or clone it elsewhere and create a symlink)
 ```bash
-cd hardware
-mkdir esp8266com
-cd esp8266com
-git clone https://github.com/esp8266/Arduino.git esp8266
+git clone https://github.com/esp8266/Arduino.git hardware/esp8266com/esp8266/
 ```
-- Download binary tools (you need Python 2.7)
+
+- This repo requires other tools (sub-modules) that we need to fetch
```

Comment:
I think we don't need that in the documentation
There's currently only one sub-sub-module `tools/sdk/lwip2/builder/lwip2-src/` which is not necessary, because it is precompiled (in `tools/sdk/lib/`).

Or you could state:
> Sub-sub modules are not necessary for a fully running repository. However they can be downloaded with the following command:

Reply:
To get a functioning toolchain you 100% need the submodules. You don't get `pyserial` or `esptool` without fetching the submodules.

Reply:
I am talking about sub-**sub**-modules

https://github.com/esp8266/Arduino/pull/5882#issuecomment-473598216

Reply:
Where are you seeing a reference to sub-sub-modules in the README?

Reply:
`git clone --recursive` takes everything, we only need one level

```
git clone ...
cd <dir>
git submodule update --init
```

`git clone --recursive` is the equivalent of
```
git clone ...
cd <dir>
git submodule update --init --recursive
```


check #5882 which is merged now